src/styles/index.scss

```diff
@@ -1,3 +1,149 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
   margin: 0;
+  padding: 0;
+}
+
+$color_dark-purple: #060b35;
+$color_white-grey: #616070;
+$color_blue: #00acdc;
+
+.page {
+
+  &__card {
+    display: flex;
+    max-width: 200px;
+    flex-direction: column;
+    background-color: #fff;
+    border: 1px solid #f3f3f3;
+    padding: 32px 16px 16px;
```

Comment:
all these styles might be applied to `card` as to BEM-block - BEM-mix with `page` block is redundant here (as page block at all)